web/satellite/static/emails/AccountAlreadyExists.html

```diff
@@ -1,599 +1,177 @@
-                                            </center>
-                                        </v:textbox>
-                                    </v:roundrect>
-                                    <![endif]-->
-                                </div>
-                            </div>
-
-                        </div>
-                        <!--[if (mso)|(IE)]></td></tr></table><![endif]-->
-                    </div>
-                </div>
-                <div style="mso-line-height-rule: exactly;line-height: 20px;font-size: 20px;">&nbsp;</div>
-                <div class="layout one-col fixed-width"
-                    style="Margin: 0 auto;max-width: 600px;min-width: 320px;width: calc(28000% - 167400px);
-                    overflow-wrap: break-word;word-wrap: break-word;word-break: break-word;">
-                    <div class="layout__inner"
-                        style="border-collapse: collapse;display: table;width: 100%; background-color: #fff;">
-                        <!--[if (mso)|(IE)]>
-                        <table align="center" cellpadding="0" cellspacing="0" role="presentation">
-                            <tr class="layout-fixed-width" style="background-color: #fff;">
-                            <td style="width: 600px" class="w560">
-                        <![endif]-->
-                        <div class="column"
-                            style="text-align: left;color: #8e959c;font-size: 14px;line-height: 21px;
-                            font-family: Helvetica, sans-serif; font-weight: 300;max-width: 600px;min-width: 320px;width: calc(28000% - 167400px);">
-                            <div style="margin: 12px 20px;">
-                                <div style="mso-line-height-rule: exactly;mso-text-raise: 4px;">
-                                    <p class="size-20"
-                                        style="Margin-top: 5px;Margin-bottom: 0;
-                                        font-family: Helvetica, sans-serif; font-weight: 300;font-size: 17px;
-                                        line-height: 26px;" lang="x-size-20">
-                                        <span>If this wasn't you, we recommend <a href="{{ .ResetPasswordLink }}" target="_blank" rel="noopener noreferrer">resetting your password</a>.
-                                            If this activity looks suspicious to you, notify our <a href="https://supportdcs.storj.io/hc/en-us" target="_blank" rel="noopener noreferrer">Support team</a>.
-                                            <br/><br/>
-                                            If your intention was to create an account on a different satellite, please select the appropriate satellite abbreviation from the drop down at
-                                                the top right of the screen when <a href="{{ .CreateAccountLink }}" target="_blank" rel="noopener noreferrer">creating an account</a>.
```

Comment:
Looks like the removed `{{ .CreateAccountLink }}` is also causing tests to fail

Reply:
yeah i think we don't need that part